gef-ui/server/controllers/unissued-facilities/index.js

```diff
@@ -203,22 +202,27 @@ const postChangeUnissuedFacility = async (req, res) => {
         canResubmitIssuedFacilities: true,
         coverDateConfirmed: true,
         unissuedToIssuedByMaker: userObj,
+        change: false,
       },
       userToken,
     });
     req.success = {
       message: `${body.facilityName} is updated`,
     };
-    req.url = `/gef/application-details/${dealId}/unissued-facilities`;
 
     // updates application with editorId
     const applicationUpdate = {
       editorId,
     };
     await api.updateApplication({ dealId, application: applicationUpdate, userToken });
 
-    // TODO: DTFS2-5227 change redirect
-    return applicationDetails(req, res);
+    // Change the redirect based on the 'change' parameter
```

Comment:
In what situation `body.change` is `true`? 
Maybe `body.change` is always `false` and we always redirect to `/gef/application-details/${dealId}/unissued-facilities`?

Reply:
From tests looks like `change` can be `true`, please ignore my comment above.

Reply:
Please let me know  below comment makes sense.
'`If change is true redirects to /gef/application-details/${dealId}`'

Reply:
Hi, comment makes sense, and I can see from the code that `If change is true redirects to /gef/application-details/${dealId}`.

Still not sure in what user journey situation `change` will be `true` and when it will be `false`. It would be nice to understand how to replicate each value (`true` `false`). Or in other words, what steps you used to test each code branch.

Reply:
I spent a lot of time yesterday looking for `body.change`, but didn't find it. Added console.log, but for me `body.change` was always `undefined`.

Most likely in this function redirectUrl is always `/gef/application-details/${dealId}/unissued-facilities`

Reply:
I added tests to check based on body.change. Are you suggesting to rewrite the logic to redirect?

Reply:
I removed above and added const redirectUrl = `/gef/application-details/${dealId}/unissued-facilities`; and added this to the test